example/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "isolatedModules": true,
+    "jsx": "react",
```

Comment:
there is also a `react-native` option here, not sure if we should use that
https://www.typescriptlang.org/docs/handbook/jsx.html